src-plugins/mscAlgorithmPaint/mscAlgorithmPaintToolBox.cpp

```diff
@@ -399,6 +399,7 @@ AlgorithmPaintToolBox::AlgorithmPaintToolBox(QWidget *parent ) :
 
     m_applyButton = new QPushButton( tr("Save") );
     m_applyButton->setToolTip(tr("Save result to the Database"));
+    m_applyButton->setObjectName("saveButton");
```

Comment:
As long as it works, can't we just keep the spirit of objectName = visual name, and once we are not able to apply this rule, we'll create another rule and apply it to every objectNames?